sonarqube/content.md

```diff
@@ -30,68 +45,119 @@ $ mvn sonar:sonar -Dsonar.host.url=http://$(boot2docker ip):9000
 
-By default, the image will use an embedded H2 database that is not suited for production.
+The images contain the SonarQube installation at `/opt/sonarqube`. You need to use bind-mounted folders to override selected files or directories :
 
-> Warning: Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
+-	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`
+-	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
+-	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
+-	`/opt/sonarqube/extensions`: plugins, such as language analyzers
 
-### Option 1: Use parameters via Docker environment variables
+### First installation
 
-The production database is configured with the following SonarQube properties used as environment variables: `sonar.jdbc.username`, `sonar.jdbc.password` and `sonar.jdbc.url`.
+Follow these steps on your first installation:
 
+1. Create a `sonarqube_home` folder and create a var env `$SQ_HOME` pointing to it:
+```console
+$ mkdir /path/to/your/filesystem/sq_home
+$ export SQ_HOME=/path/to/your/filesystem/sq_home
+```
+2. Initialize SQ_HOME folder tree with `--init`. This will initialize the default configuration, copy embedded plugins, and prepare the data folder:
+```console
+$ docker run --rm \
+ -v $SQ_HOME/conf:/opt/sonarqube/conf \
+ -v $SQ_HOME/extensions:/opt/sonarqube/extensions \
+ -v $SQ_HOME/data:/opt/sonarqube/data \
+ sonarqube --init
+```
+3. Configure sonar.properties & follow the normal first installation procedure (TODO add a link).
+4. Run the image:
 ```console
 $ docker run -d --name sonarqube \
     -p 9000:9000 \
-    -e sonar.jdbc.username=sonar \
-    -e sonar.jdbc.password=sonar \
-    -e sonar.jdbc.url=jdbc:postgresql://localhost/sonar \
+    -v $SQ_HOME/conf:/opt/sonarqube/conf \
+    -v $SQ_HOME/extensions:/opt/sonarqube/extensions \
+    -v $SQ_HOME/logs:/opt/sonarqube/logs \
+    -v $SQ_HOME/data:/opt/sonarqube/data \
     sonarqube
 ```
+5. enjoy
 
-Use of the environment variables `SONARQUBE_JDBC_USERNAME`, `SONARQUBE_JDBC_PASSWORD` and `SONARQUBE_JDBC_URL` is deprecated, and will stop working in future releases.
 
-More recipes can be found [here](https://github.com/SonarSource/docker-sonarqube/blob/master/recipes.md).
+## Upgrade SonarQube
+Follow these steps to upgrade SonarQube:
 
-You can pass `sonar.` configuration properties as Docker environment variables, as demonstrated in the example above for database configuration.
+1. Create a new `sonarqube_home` folder, and update the var env `$SQ_HOME` pointing to it. Backup the old one:
```

Comment:
I'm assuming the following commands show how to update $SQ_Home to point to the new folder.  If this is the case, I would make this two steps like this:

1. Create a new `sonarqube_home` folder and backup your old folder.

2. Update the var env `$SQ_HOME` to point to your new folder:

> $ mkdir /path/to/your/filesystem/sq_home81
$ export SQ_HOME=/path/to/your/filesystem/sq_home81

Reply:
agree. Please feel free to directly suggest changes :)